server/hedley/modules/custom/hedley_reports/scripts/generate-well-child-completion-data.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Generates completion data for different types of encounters.
+ *
+ * Execution: drush scr
+ *   profiles/hedley/modules/custom/hedley_reports/scripts/generate-well-child-completion-data.php.
+ */
+
+if (!drupal_is_cli()) {
+  // Prevent execution from browser.
+  return;
+}
+
+// Get the last node id.
+$nid = drush_get_option('nid', 0);
+
+// Get the number of nodes to be processed.
+$batch = drush_get_option('batch', 50);
+
+// Flag to generate data only if it was not generated already.
+$exclude_set = drush_get_option('exclude_set', FALSE);
+
+// Get allowed memory limit.
+$memory_limit = drush_get_option('memory_limit', 500);
+
+$type = 'individual_participant';
+$base_query = new EntityFieldQuery();
+$base_query
+  ->entityCondition('entity_type', 'node')
+  ->entityCondition('bundle', $type)
+  ->fieldCondition('field_encounter_type', 'value', 'well-child')
```

Comment:
Do we need to exclude deleted here?

Reply:
As we load participants here, indeed we need to add the tag. Nice catch!